scalability_and_performance/understanding-node-observability-operator.adoc

```diff
@@ -0,0 +1,363 @@
+----
+
+. Verify the install plan status:
++
+[source,terminal]
+----
+$ oc -n node-observability-operator get ip <install_plan_name> -o yaml | yq .status.phase'
+----
++
+`<install_plan_name>` is the install plan name that you obtained from the output of the previous command.
+
++
+.Example output
+[source,terminal]
+----
+COMPLETE
+----
+
+. Verify that the Node Observability Operator is up and running:
++
+[source,terminal]
+----
+$ oc get deploy -n node-observability-operator
+----
+
++
+.Example output
+[source,terminal]
+----
+NAME                                            READY   UP-TO-DATE  AVAILABLE   AGE
+node-observability-operator-controller-manager  1/1     1           1           40h
+----
+
+=== Installing the Node Observability Operator using the web console
+You can install the Node Observability Operator from the {product-title} web console.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have access to the {product-title} web console.
+
+.Procedure
+
+. Log in to the OpenShift Container Platform web console.
+. In the Administrator's navigation panel, expand *Operators* → *OperatorHub*.
+. In the *All items* field, enter `node-observability-operator` and select the Node Observability Operator tile.
+. Click *Install*.
+. On the *Install Operator* page:
+.. *Update channel*: Select *alpha*.
+.. *Installation mode*: Select *A specific namespace on the cluster*.
+.. *Installed Namespace*: Choose *node-observability-operator* from the list.
+.. *Update approval*: Select *Automatic*.
+.. Click *Install*.
+
+.Verification
+. In the Administrator's navigation panel, expand *Operators* → *Installed Operators*.
+. Verify that the Node Observability Operator is listed in the Operators list.
+
+== Creating the Node Observability custom resource
+Before you run profiling queries, you must create a `NodeObservability` custom resource (CR).
+
+When you run the the `NodeObservability` CR, it creates the necessary machine config and machine config pool CRs to enable the CRI-O profiling on selected worker nodes.
+
+[NOTE]
+====
+Kubelet profiling is enabled by default.
+====
+
+.Prerequisites
+* The Node Observability Operator has been installed.
+* You have access to the cluster with `cluster-admin` privileges.
+
+.Procedure
+
+. Log in to the {product-title} CLI as a user with the `cluster-admin` role:
++
+[source,terminal]
+----
+$ oc login -u kubeadmin https://<HOSTNAME>:6443
+----
+
+. Create a new project named `node-observability-operator`:
++
+[source,terminal]
+----
+$ oc project node-observability-operator
+----
+
+. Create a CR file named `nodeobservability.yaml` that contains the following text:
++
+[source,yaml]
+----
+    apiVersion: nodeobservability.olm.openshift.io/v1alpha1
+    kind: NodeObservability
+    metadata:
+      name: cluster <1>
+    spec:
+      labels:
+      type: crio-kubelet
+----
+<1> You must specify the name as `cluster` because it is applied cluster-wide.
```

Comment:
@alebedev87 How about this rewrite ? wdyt ?

<1> You must specify the name as `cluster` because there can be only one `NodeObservability` CR per cluster.